gen/jsonschema/schemas/Bundle.schema.json

```diff
@@ -65,6 +65,10 @@
                     "$ref": "#/definitions/dev.sigstore.common.v1.X509CertificateChain",
                     "additionalProperties": false
                 },
+                "certificate": {
```

Comment:
would x509certificate be more consistent or is it just kind of redundant?

I guess this naming is in line with X509CertificateChain.certificates?

Reply:
Yeah, I was thinking the same. `x509_certificate` would be consistent with the other variant, but it feels a bit redundant. I have no strong opinions here 🙂

Reply:
just leave as is unless @haydentherapper or @kommendorkapten chime in?

Reply:
What if we add support for SSH certificates? 😄 

No but really, leaving off `x509` is fine with me. We probably should have named the other field `certificate_chain` as x509 is a bit too in the weeds.